workspace/utilities/lib/site-title.xsl

```diff
@@ -5,17 +5,7 @@
 		<xsl:param name="title" />
 		<xsl:param name="config-title" select="$config/title" />
 		<xsl:param name="params-title" select="$params/website-name" />
-		<xsl:call-template name="site-title-default">
-			<xsl:with-param name="title" select="$title" />
-			<xsl:with-param name="config-title" select="$config/title" />
-			<xsl:with-param name="params-title" select="$params/website-name" />
-		</xsl:call-template>
-	</xsl:template>
 
-	<xsl:template name="site-title-default">
-		<xsl:param name="title" />
-		<xsl:param name="config-title" select="$config/title" />
-		<xsl:param name="params-title" select="$params/website-name" />
```

Comment:
By removing the default version, we are loosing an extension point, and the default version can not be used as is elsewhere. I wonder if this is really what we want ?

Reply:
@fhamon  ? @dominic-blain  ?